sklearn/tests/test_calibration.py

```diff
@@ -275,6 +275,34 @@ def multiclass_brier(y_true, proba_pred, n_classes):
     assert calibrated_brier < 1.1 * uncalibrated_brier
 
 
+def test_calibration_zero_probability():
+    class FakeClassifier():
+        def __init__(self, n_classes):
+            self.classes_ = n_classes
+
+        def predict_proba(self, X):
+            return np.empty_like(X)
+
+    class FakeCalibrator():
```

Comment:
```suggestion
    class ZeroCalibrator():
```

to make it explicit that this test in concerned with calibrators that predict zero for all the classes at once.